rust_src/src/windows.rs

```diff
@@ -2034,3 +2035,15 @@ pub fn window_pixel_height_before_size_change(window: LispWindowValidOrSelected)
 }
 
 include!(concat!(env!("OUT_DIR"), "/windows_exports.rs"));
+
+/// Run `window-configuration-change-hook' for FRAME.
+/// If FRAME is omitted or nil, it defaults to the selected frame.
+#[lisp_fn(min = "0")]
+pub fn run_window_configuration_change_hook(frame: Option<LispObject>) -> LispObject {
```

Comment:
```suggestion
pub fn run_window_configuration_change_hook(frame: LispFrameLiveOrSelected) {
```

Wherever you see `decode_live_frame` you can use `LispFrameLiveOrSelected` instead. Then in the function:
`let frame_ref: LispFrameRef = frame.into();` Now you have a Rust `LispFrameRef` to use.